cmd/ci_status_test.go

```diff
@@ -103,3 +103,30 @@ deploy: deploy10                       - success`)
 
 	assert.Contains(t, out, "Pipeline Status: success")
 }
+
+// Test_ciStatusMRPassed tests a behavior that is not documented in GitLab's
+// API docs. So this is basically to make sure we don't get hit by a
+// future/unnotified API change.
+func Test_ciStatusMRPassed(t *testing.T) {
+	t.Parallel()
+	repo := copyTestRepo(t)
+	cmd := exec.Command(labBinaryPath, "ci", "status", "--merge-request", "968")
```

Comment:
> I realized that --passed is ignored if --merge-request is given.
> I'm not sure if there's an easy fix. If not, we better fail on this combination of arguments.

Hmm.. the `--passed` option is ignored when `--merge-request` is *not* given, was it a typo?
The `onlyPassed` flag is checked directly within `if forMR {...}` conditional :P. Or am I missing something?

> At least 600 is interpreted as branch name (and not as MR ID or pipeline ID), so it's weird that it's a number.

I *think* this example should have the `--merge-request`, otherwise it indeed is a strange example. However, not an impossible one. People can create branches with numbers right? If a team is used to work with external ticket systems or is new to branch naming conventions, it's not that difficult to find cases where the branch name matches the ticket number. (that's something I see somewhat frequently in the team I work at RedHat hahah. Sad? Yes! But still happens).